go/vumitools/api.py

```diff
@@ -354,9 +353,10 @@ def _update_tag_data_for_acquire(self, user_account, tag):
         # The batch we create here gets added to the tag_info and we can fish
         # it out later. When we replace this with proper channel objects we can
         # stash it there like we do with conversations and routers.
-        yield self.api.mdb.batch_start([tag], user_account=user_account.key)
+        yield self.api.get_batch_manager().batch_start(
```

Comment:
Minor thing... maybe put the batch manager in a variable here since `get_batch_manager()` creates a new manager (although they're pretty lightweight).